test/unit/util/test_custom_templates.py

```diff
@@ -0,0 +1,59 @@
+"""Test custom template configuration and rendering."""
+
+from galaxy.util.custom_templates import template
+
+TEMPLATE_RELPATH = "mail/activation-email.txt"
```

Comment:
you can refactor the `users.py` module and expose the values of the template paths as constants. Then you can import the constant from that module; as a result, that module will be the single source of truth for the value of those paths (no duplication!). So, instead of `TEMPLATE_RELPATH`, you'll have something like `from galaxy.managers.users import ACTIVATION_EMAIL_TEMPLATE_PATH_TXT as template_relpath` (or whatever you want to call it).

Reply:
Done

Reply:
Sorry - my mistake! This results in a test that's part of the util package importing something from a manager module, which is incorrect (managers import from utils, but not the other way round). That's what's causing the sqlalchemy module not found error (and the util package shouldn't know about sqlalchemy). So, just leave that path hardcoded as you had it in your previous version, before my suggestion broke everything..

Reply:
No worries, sorted! It's a complicated ecosystem :joy: